RELEASE-NOTES.md

```diff
@@ -4,6 +4,95 @@ section: overview
+- **Page header:** 
+  - Removed redundant role="banner" attribute ([#3499](https://github.com/patternfly/patternfly-react/pull/3499))
+- **Page layout:** 
+  - Fixed page layout imports for use with code sandbox.([#3461](https://github.com/patternfly/patternfly-react/pull/3461))
+- **Pagination:**
+  - Disabled navigation input if page is equal to zero ([#3472](https://github.com/patternfly/patternfly-react/pull/3472))
+- **Select:** 
+  - Add the type prop to the select toggle button ([#3495](https://github.com/patternfly/patternfly-react/pull/3495))
+  - Allowed custom option data comparison ([#3491](https://github.com/patternfly/patternfly-react/pull/3491))
+  - Updated so clear button appears with any type ahead input ([#3502](https://github.com/patternfly/patternfly-react/pull/3502))
+  - Set a unique id to the select toggle type ahead input ([#3529](https://github.com/patternfly/patternfly-react/pull/3529))
+- **Spinner** 
+ - Promote Spinner from experimental ([#3516](https://github.com/patternfly/patternfly-react/pull/3516))
+
+### Catalog view extension
+- **Catalog tile:** Removed truncation fade and fixed length ([#3378](https://github.com/patternfly/patternfly-react/pull/3378))
+
+### Table
+  - Fixed issue 3559, for table we now default to a div instead of anchor([#3567](https://github.com/patternfly/patternfly-react/pull/3567))
+  - Made IAction onClick optional ([#3558](https://github.com/patternfly/patternfly-react/pull/3558))
+
+### Other
+- **Ci:** 
+  - Stopped caching stale doc builds ([#3582](https://github.com/patternfly/patternfly-react/pull/3582))
+  - Added coverage reporting for cypress tests ([#3295](https://github.com/patternfly/patternfly-react/pull/3295))
+- **Chore:** 
+  - Used absolute import paths for react core in pf4 packages. ([#3525](https://github.com/patternfly/patternfly-react/pull/3525))
+  - Fixed homepage URL ([#3522](https://github.com/patternfly/patternfly-react/pull/3522))
+  - Removed listing of icons while under construction ([#3485](https://github.com/patternfly/patternfly-react/pull/3485))
+  - Use direct paths to react-icons in production build ([#3448](https://github.com/patternfly/patternfly-react/pull/3448))
+- **Demos:** 
+  - Upped version of gatsby-theme-patternfly-org ([#3562](https://github.com/patternfly/patternfly-react/pull/3562))
+- **Docs:** 
+  - Added a link to icons docs ([#3430](https://github.com/patternfly/patternfly-react/pull/3430))
+- **Extensions:** 
+  - Moved extensions to 'Extensions' in side nav category ([#3453](https://github.com/patternfly/patternfly-react/pull/3453))
+- **Integration:**
+  - Fixed selector in table simple actions ([#3584](https://github.com/patternfly/patternfly-react/pull/3584))
+- **Linting:** 
+  - Made Eslint fixes for *.md file ([#3493](https://github.com/patternfly/patternfly-react/pull/3493))
+  - Made initial eslint fixes for react-core-style-system *.md file ([#3505](https://github.com/patternfly/patternfly-react/pull/3505))
```

Comment:
Double-check your spacing around here, the Github editor stops syntax highlighting due to some inconsistent spacing I think.